robottelo/config/validators.py

```diff
@@ -35,10 +36,20 @@
         Validator('server.ssh_username', default='root'),
         Validator('server.ssh_password', default=None),
         Validator('server.verify_ca', default=False),
-        Validator('server.is_ipv6', is_type_of=bool, default=False),
+        Validator(
+            'server.network_type',
+            is_in=NetworkType.list_values(),
+            default=NetworkType.IPV4.value,
+        ),
```

Comment:
If you use `cast=NetworkType` won't it raise a `ValueError` if the value is invalid? That way you both have a strong type and validation. It also allows you to drop the `list_values()` implementation.

Reply:
You are right! Good point. I will update it.